third_party/terraform/resources/resource_sql_database_instance.go.erb

```diff
@@ -1051,6 +1084,15 @@ func resourceSqlDatabaseInstanceUpdate(d *schema.ResourceData, meta interface{})
 		return err
 	}
 
+	if r, ok := d.GetOk("restore_backup_context"); ok {
```

Comment:
To support restoring from a backup during Update, we'll have to trigger when changes are made to `restore_backup_context` other than the field just being removed

Reply:
Can you comment that inline?